network-api/networkapi/wagtailpages/pagemodels/buyersguide/products.py

```diff
@@ -1617,6 +1619,14 @@ def get_context(self, request, *args, **kwargs):
         context['template_cache_key_fragment'] = f'pni_home_{request.LANGUAGE_CODE}'
         return context
 
+    def find_editorial_content_index(self):
+        BuyersGuideEditorialContentIndexPage = apps.get_model(
+            app_label='wagtailpages',
+            model_name='BuyersGuideEditorialContentIndexPage',
+        )
+        indexes = BuyersGuideEditorialContentIndexPage.objects.descendant_of(self)
+        return indexes.first()
+
```

Comment:
**Praise**

I like that you've added tests for returning `None` on this :ok_hand: